frontend-student/src/components/allcourses/FilterControls.jsx

```
@@ -0,0 +1,115 @@
+            : 'bg-gray-100 text-gray-700 hover:bg-gray-200'
+        }`}
+        whileHover={{ scale: 1.05 }}
+        whileTap={{ scale: 0.95 }}
+      >
+        <span className='mr-2'>Rating</span>
+        <svg
+          className='w-5 h-5'
+          fill='none'
+          viewBox='0 0 24 24'
+          stroke='currentColor'
+        >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)